### ISSUE TITLE
RadAutoCompleteTextView does not have exposed events for DisplayMode = "Plain"

### ISSUE THREAD

**Issue by NickIliev** [telerik/nativescript-ui-feedback#149]
Valid for both **iOS** and **Android**.
At his very moment, the only [events](http://docs.telerik.com/devtools/nativescript-ui/Controls/NativeScript/AutoCompleteTextView/autocomplete-events) exposed for RadAutoCompleteTextView are applicable when the **displayMode** is set to **Tokens**.

When the display mode is set to **Plain** there are no events that can be handled.
Requested by t,.1102467.

Possible workaround provided for Android is to override the native event listener
e.g.
```
    var autocmp = args.object

    console.log("autocmp: " + autocmp)

    var nativeTField = autocmp.android.getTextField();
    console.log("nativeTextField: " + nativeTField) // android.widget.EditText

    nativeTField.addTextChangedListener(new android.text.TextWatcher({

        afterTextChanged(s) {
            console.log("afterTextChanged");
            console.log(s);
        },
        beforeTextChanged(s, start, count, after) {
            console.log("beforeTextChanged");
            console.log(s);
            console.log(start);
            console.log(count);
            console.log(after);
        },
        onTextChanged(s, start, before, count) {
            console.log("onTextChanged");
            console.log(s);
            console.log(start);
            console.log(before);
            console.log(count);
        }
    }));
}

```

**Comment by NickIliev** [comment]
+1 from t.1109497

**Comment by randy-johnson** [comment]
Any updates on this?

**Comment by VladimirAmiorkov** [comment]
@randy-johnson We have addressed this in our internal "next" version and it will be available in the next official release 3.0.0. 

You can test it right now by using our "next" tag version `npm install nativescript-telerik-ui-pro@next`, note that this is an version which is updated multiple times on daily basis from our internal development branch and includes all of the latest fixes and new features, meaning it is not suitable for production code but rather for testing the fore mentioned fixes/features.

You can use the following events:

- `tokenRemoved` - triggered whenever a token is removed.
- `tokenSelected` - triggered whenever a token is selected.
- `tokenDeselected` - triggered whenever a token is deselected.
- `didAutoComplete` - triggered whenever an item from the suggestions list is selected.
- `suggestionViewBecameVisible` - triggered whenever the suggestion view is shown.

**Comment by randy-johnson** [comment]
Any ideas when the next version will be production ready?

**Comment by VladimirAmiorkov** [comment]
The 2.0.0 release is planned to be released by the end of thus month.

**Comment by VladimirAmiorkov** [comment]
@randy-johnson Forgive my last response, the mentioned feature will be available in the **3.0.0** not **2.0.0**, it was a simple typo on my end.

**Comment by tgpetrov** [comment]
As promised these events were released with 3.0.0. 
[Here](http://docs.telerik.com/devtools/nativescript-ui/Controls/NativeScript/AutoCompleteTextView/autocomplete-events)'s the documentation about them.

**Comment by Dobermensch** [comment]
I'm using displaymode Plain and the events tokenAdded,tokenRemoved,tokenSelected,tokenDeselected are not firing. In the docs, the example for [events](https://github.com/ProgressNS/nativescript-ui-samples/blob/master/autocomplete/app/examples/events/autocomplete-events-model.ts) is using displayMode Tokens
